test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildADesktopExe.cs

```diff
@@ -68,19 +68,18 @@ public void It_builds_with_inferred_platform_target(string runtimeIdentifier, st
                 .WithSource()
                 .Restore("", $"/p:RuntimeIdentifier={runtimeIdentifier}");
 
-            var buildCommand = new BuildCommand(Stage0MSBuild, testAsset.TestRoot);
-            buildCommand
+            var getValuesCommand = new GetValuesCommand(Stage0MSBuild, testAsset.TestRoot,
+                "net46", "PlatformTarget", GetValuesCommand.ValueType.Property);
+
+            getValuesCommand
                 .Execute($"/p:RuntimeIdentifier={runtimeIdentifier}")
                 .Should()
                 .Pass();
 
-            var assemblyPath = Path.Combine(buildCommand.GetOutputDirectory("net46").FullName, "DesktopMinusRid.exe");
-            var assemblyInfo = AssemblyInfo.Get(assemblyPath);
-            assemblyInfo
+            getValuesCommand
+                .GetValues()
                 .Should()
-                .Contain(
-                    "AssemblyDescriptionAttribute",
-                    $"PlatformTarget={expectedPlatformTarget}");
+                .BeEquivalentTo(expectedPlatformTarget);
```

Comment:
What does equivalent mean here? Isn't the a plain Be()? It should be exactly that.